cache_system/src/backend/policy/lru.rs

```diff
@@ -653,10 +657,11 @@ where
         // maybe clean from pool
         {
             let mut inner = self.inner.lock();
-            if let Some((consumption, _last_used)) = inner.last_used.remove(&k) {
+            if let Some((consumption, last_used_already)) = inner.last_used.remove(&k) {
```

Comment:
```suggestion
            if let Some((consumption, last_used_previously)) = inner.last_used.remove(&k) {
```

Minor naming suggestion I would have found easier to understand